src/pages/AutoDeposit/AutoDepositLandingPage.vue

```diff
@@ -137,33 +136,49 @@ export default {
 		result({ data }) {
 			// Extract page content from contentful
 			const pageEntry = data.contentful?.entries?.items?.[0] ?? null;
-			this.pageData = pageEntry ? processPageContentFlat(pageEntry) : null;
+			this.pageData = pageEntry ? processPageContent(pageEntry) : null;
 
-			this.isMonthlyGoodSubscriber = _get(data, 'my.autoDeposit.isSubscriber', false);
-			this.hasAutoDeposits = !!_get(data, 'my.autoDeposit') && !this.isMonthlyGoodSubscriber;
-			const legacySubs = _get(data, 'my.subscriptions.values', []);
+			this.isMonthlyGoodSubscriber = data?.my?.autoDeposit?.isSubscriber ?? false;
+			this.hasAutoDeposits = data?.my?.autoDeposit ?? false;
+
+			const legacySubs = data?.my?.subscriptions?.values ?? [];
 			this.hasLegacySubscription = legacySubs.length > 0;
 		},
 	},
 	computed: {
+		contentGroups() {
+			return this.pageData?.page?.pageLayout?.contentGroups ?? [];
+		},
 		canDisplayForm() {
 			return !this.isMonthlyGoodSubscriber && !this.hasAutoDeposits && !this.hasLegacySubscription;
 		},
+		faqContentGroup() {
+			return this.contentGroups?.find(({ type }) => {
+				return type ? type === 'frequentlyAskedQuestions' : false;
+			});
+		},
+		ctaContentGroup() {
+			return this.contentGroups?.find(({ key }) => {
+				return key ? key === 'auto-deposit-cta' : false;
```

Comment:
Can we target this based on a type instead? Same with the key matching below?

Reply:
These contentGroups don't have a type.

Reply:
Ok I see they would need a "type" and prob some deeper content refactor to accommdate that.